### ISSUE TITLE
Location of Database for PGBackup

### ISSUE THREAD

**Issue by Bestmixtape** [elliotmatson/Docker-Davinci-Resolve-Project-Server#1]
Hi,

For the Docker-Compose, do you need to add this into volumes section for PG backup?

- "/resolve-db:/var/lib/postgresql/data"

I only noticed an issue when I updated my database to 13 and my backups were only 1kb. I looked at my docker container in Portainer, and Portainer automatically assigned /var/lib/postgresql/data to an internal created storage instead of /resolve-db.  

I didn't have backup issues on the previous versions.

**Comment by elliotmatson** [comment]
No, you shouldn't need to. pgbackups accesses the database over the network created for this docker-compose stack, not the resolve-db volume.

How did you update your database? Is it functioning properly in Resolve? If you make a backup in Resolve, how big is it?

**Comment by Bestmixtape** [comment]
Thanks for the quick reply Elliott. From your question, I realized where I made the mistake. Instead of updating, I did a clean install and copied over the old projects to the new database. 

I noticed when creating the new database, it didn't match exactly what I put in the Docker compose. 

It's running well now, and thank you for creating this Docker compose. 

Would love to see an addition of pgadmin to edit/delete corrupted projects. I'll see if I can figure that out.
